homeassistant/components/ipp/__init__.py

```diff
@@ -30,6 +31,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             tls=entry.data[CONF_SSL],
             verify_ssl=entry.data[CONF_VERIFY_SSL],
         )
+
+        # Update coordinator in an async context.
+        await coordinator.update_more_info_url()
+
         hass.data[DOMAIN][entry.entry_id] = coordinator
```

Comment:
This should be done after first refresh, since that is weekend we know printer is available.